.github/workflows/ci.yaml

```diff
@@ -227,6 +227,19 @@ jobs:
           # To run the CI with debug informations, add --info
           ./gradlew jacocoTestReport --parallel --build-cache
 
+      - name: Generate Sarif Lint Report
+        shell: bash
+        run: |
+          # To run the CI with debug informations, add --info
+          ./gradlew lintDebug --parallel --build-cache
+      
+      - name: Upload lint sarif
+        uses: github/codeql-action/upload-sarif@v2
+        if: success() || failure()
```

Comment:
why this `if` clause here but neither before nor after ?

Reply:
No particular reason as the order doesn't change the workflow

Reply:
instead of `success() || failure()` why not use `always()`?